Changes in inet.h

I use make install on libctru and it didn't throw up any errors until.

However, when using libctru to compile other projects, the changes in (
https://github.com/smealum/ctrulib/commit/28b377511827af81d4b9e6b41bb9bc7ab289566e) cause errors:
`c:/devkitPro/libctru/include/arpa/inet.h:34:53: error: expected ',' or '...' before 'src'
  const char *inet_ntop(int af, const void *restrict src, char *restrict dst, socklen_t size);`

`c:/devkitPro/libctru/include/arpa/inet.h:35:52: error: expected ',' or '...' before 'src'
  int        inet_pton(int af, const char *restrict src, void *restrict dst);`

Manually removing lines 34 and 35 completes the compile. 
Not tested the completed compile to see if it is functional yet.

--- a/libctru/include/arpa/inet.h
+++ b/libctru/include/arpa/inet.h
@@ -31,8 +31,8 @@ extern "C" {
 	int       inet_aton(const char *cp, struct in_addr *inp);
 	char*     inet_ntoa(struct in_addr in);
 
-	const char *inet_ntop(int af, const void *restrict src, char *restrict dst, socklen_t size);
-	int        inet_pton(int af, const char *restrict src, void *restrict dst);
+	const char *inet_ntop(int af, const void * src, char * dst, socklen_t size);
+	int        inet_pton(int af, const char * src, void * dst);
 
 #ifdef __cplusplus
 }

